src/components/uyghur/people/UyghurActivistsCards.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+import {
+  Card,
+  Divider,
+  Image,
+} from 'semantic-ui-react';
+
+class UyghurActivistsCards extends Component {
```

Comment:
I appreciate you actually referring to my cards, and using the same practice of separating ActivistsCards like I did. But it just made me realize I forgot to separate out the Journalists and other people like intended. So I am going to go ahead and fix that all in this PR 😄